### PR TITLE
HTML/CSS updates based on updated page design

### PR DESCRIPTION
**Pull request by jgiardino**
HTML and CSS were updated to implement the UI design for Edit Recipe as shown here: https://trello.com/c/89YOu8XU/308-3-ui-design-edit-recipe-page-layout

Part of these updates include switching out the older PF list view pattern with the newer one, which resulted in updates to the list views for Revisions and Compositions as well.

Additional updates are pending, related to addressing the scrolling on the Edit Recipe page.

### CODE REVIEW

**Comment by jgiardino** [comment]
[(5) [UI dev] Edit Recipe page layout (js)](https://trello.com/c/uAjTBF0q/361-5-ui-dev-edit-recipe-page-layout-js)

File: components/ListView/ComponentInputs.js [hunk 1]
```diff
@@ -2,133 +2,155 @@ import React from 'react';
...
+      // hide tooltip for other list items
+      if ($(event.currentTarget).hasClass('list-pf-container')) {
+        $('.list-pf-container[data-toggle="tooltip"]').not(event.target).tooltip('hide');
+      }
+      // hide tooltip for component list item if hovering over an action
+      if ($(event.currentTarget).parent('.list-pf-actions').length) {
+        this.hideTooltip('parent');
       }
+      $(event.currentTarget).tooltip('show');
+      console.log('show', event.currentTarget);
```
Comment (henrywang) [inline components/ListView/ComponentInputs.js]: Could you please remove this line of debug code? This debug code will generate console output in browser.

File: components/ListView/ComponentInputs.js [hunk 2]
```diff
@@ -2,133 +2,155 @@ import React from 'react';
...
-      event.preventDefault();
-      event.stopPropagation();
+  bindTooltipMouseleave() {
+    $('.cmpsr-list-inputs').on('mouseleave', '[data-toggle="tooltip"]', event => {
+      this.hideTooltip(event.currentTarget);
+      if ($(event.currentTarget).parent('.list-pf-actions').length) {
+        $(event.currentTarget).parents('.list-pf-container').tooltip('show');
+        console.log('show parent of action');
+      }
+      console.log('hide', event.currentTarget);
```
Comment (henrywang) [inline components/ListView/ComponentInputs.js]: And remove this debug code. The same reason as above comment.

**Comment by atodorov** [comment]
Hi @jgiardino,
can you resolve the conflicts so we can merge this PR. From what I understand it is blocking further work and it's been 11 days since this has been opened.

**Pull request merged**